src/shapes/1d/gaussian/Gaussian.ts

```diff
@@ -45,142 +45,148 @@ export interface IGetAreaGaussianOptions {
   sd?: number;
 }
 
-export class Gaussian extends Shape1DClass {
+export interface IGaussianClass {
```

Comment:
Any reason for duplicating the interface in each class? This defeats the utility of using an interface.